source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+Scopo
+-------
+
+Lo scopo principale di tutte le conferenze e user groups che aderiscono a questo Codice di Condotta è di essere il più inclusivi possibile rispetto al maggior numero di contributori, provenienti dai piu' eterogenei e variegati backgrounds possibili. Per questo, ci impegniamo a fornire un ambiente amichevole, sicuro e accogliente, indipendentemente dal sesso, orientamento sessuale, disabilità fisiche e/o mentali, etnia, stato socio-economico e preferenze religiose (o mancanza di tali).
+
+Questo Codice di Condotta delinea cosa ci aspettiamo da tutti i partecipanti delle nostre communities, cosí come le conseguenze per eventuali comportamenti inaccettabili.
```

Comment:
I would also translate communities with "comunità". It has the same meaning and it's used often in the same context in Italian.